src/brutus-module-math/brutus_module_math/nlCalc.py

```diff
@@ -1,76 +1,93 @@
+units = [
+    "zero",
+    "one",
+    "two",
+    "three",
+    "four",
+    "five",
+    "six",
+    "seven",
+    "eight",
+    "nine",
+    "ten",
+    "eleven",
+    "twelve",
+    "thirteen",
+    "fourteen",
+    "fifteen",
+    "sixteen",
+    "seventeen",
+    "eighteen",
+    "nineteen"
+]
+tens = [
+    "",
+    "",
+    "twenty",
+    "thirty",
+    "forty",
+    "fifty",
+    "sixty",
+    "seventy",
+    "eighty",
+    "ninety"
+]
+scales = [
+    "hundred",
+    "thousand",
+    "million",
+    "billion",
+    "trillion"
+]
+
+
 def calculate(string):
+    string = fixSpecialCases(string)
     string = convertToNumbers(string)
     string = convertToSymbols(string)
+    # string = fixNegatives(string)
```

Comment:
I would remove this line. The code is versioned so we can always look back to see what we had before.